graylog2-server/src/main/java/org/graylog2/periodical/ClusterHealthCheckThread.java

```diff
@@ -35,18 +36,24 @@
     private NotificationService notificationService;
     private final InputRegistry inputRegistry;
     private final NodeId nodeId;
+    private boolean isCloud;
 
     @Inject
     public ClusterHealthCheckThread(NotificationService notificationService,
                                     InputRegistry inputRegistry,
-                                    NodeId nodeId) {
+                                    NodeId nodeId,
+                                    @Named("is_cloud") boolean isCloud) {
         this.notificationService = notificationService;
         this.inputRegistry = inputRegistry;
         this.nodeId = nodeId;
+        this.isCloud = isCloud;
     }
 
     @Override
     public void doRun() {
+        if (isCloud) {
+            return;
```

Comment:
I meant moving the log statement to debug. Now there is no explanation why we do this. :wink:

Reply:
Thanks. This was the original intention, but somehow I left it at info. Makes sense, I see that now.